### PR TITLE
App deck

### PR DESCRIPTION
**Pull request by RaginaPhalangae**
Fixes: #57 
I made a deck of apps like Zomato, Facebook, etc. I hope you'll like this deck as well and merge it,
hoping for a positive response.

### CODE REVIEW

File: decks.js [hunk 1]
```diff
@@ -1379,6 +1379,105 @@ echo $message;`,
...
+              name: "TikTok",
+              content: {
+                  secondary: "✔️💬"
+              },
+              hint: "It was initially named Musical.ly.",
+              matchValue:"It was founded by Zhang Yiming.",
+              seenHint: false
+          },
+          {
+              name: "Disney+Hotstar",
```
Comment (N-Shar-ma) [inline decks.js:1424]: Can you change this to just Hotstar? The app is only named hotstar and the emojis too represent only that

File: decks.js [hunk 2]
```diff
@@ -1379,6 +1379,105 @@ echo $message;`,
...
+              hint: "Tagline: Swipe Right.",
+              matchValue: "It was launched by Sean Rad.",
+              seenHint: false
+          },
+          {
+              name: "Gmail",
+              content: {
+                  secondary: "G📩"
+              },
+              hint: "It introduced a web development technique named Ajax.",
```
Comment (N-Shar-ma) [inline decks.js:1455]: Can you rephrase the hint to be "It made mainstream a web development technique called AJAX" since AJAX was introduced before Gmail but this was the 1st time it was used in something as impactful as Gmail.

**Review by N-Shar-ma — APPROVED** [review]
The deck looks great @RaginaPhalangae! Just make the 2 requested changes and this should be good to merge!

File: decks.js (hunk 1 above)
Comment (RaginaPhalangae) [reply decks.js:1424]: yeah sure!

File: decks.js (hunk 2 above)
Comment (RaginaPhalangae) [reply decks.js:1455]: sure...thank you for the information

**Comment by RaginaPhalangae** [comment]
@N-Shar-ma I made the changes you suggested. Thank you for your suggestions, and I hope that it is now good to merge.

**Comment by N-Shar-ma** [comment]
All's good now, thank you for the fun contribution @RaginaPhalangae! 😄
